frontend/README.md

```diff
@@ -28,7 +28,7 @@ There are two configurations needed for the frontend to run, those are passed as
 ### Modify Environment Variables
 
 * Follow the guide on how to run the Backend locally.  You can find the README [here](../api/README.md)
-* Once the Backend has started, you will want to get the port the Backend uses *(local port may change depending upon the IDE being used)* and modify the `VUE_APP_BACKEND` environment variable value in the `env.development` file.  See below:
+* Once the Backend has started, you will want to get the port the Backend uses *(local port may change depending upon the IDE being used)* and modify the `VUE_APP_BACKEND` environment variable value in the `.env.development.local` file.  See below:
```

Comment:
Should this just be `.env.local` vs `.env.development.local` ?

Reply:
I think it doesn't matter in this case, because for local development we are in the `development` mode, but we could change it to `.env.local`